test/functional/test_framework/authproxy.py

```diff
@@ -102,6 +102,7 @@ def _request(self, method, path, postdata):
                    'User-Agent': USER_AGENT,
                    'Authorization': self.__auth_header,
                    'Content-type': 'application/json'}
+        path = urllib.parse.quote(path)
```

Comment:
I'm not sure this is the correct place to do URL-quoting. Wouldn't the best place to this be where the path is generated?

Reply:
so this would be `get_wallet_rpc(self, wallet_name)`

Reply:
Indeed, we woulndn't want to double-quote in case the path is already quoted.